content/build-tool.md

```diff
@@ -79,7 +79,16 @@ One difference between pre-published packages and local app packages is that the
 
 <h4 id="npm-adding">Adding packages to your app</h4>
 
-Meteor 1.3 will have seamless integration with NPM, and you will be able to simply `npm install` these packages into your app directory. Until then, the easiest way to use NPM packages in your app is [`meteorhacks:npm`](https://atmospherejs.com/meteorhacks/npm).
+To install an NPM package into your app, you can simply run NPM's standard install command:
+
+```
+npm install --save-dev <package name>
+```
+
+XXX: Is it `meteor npm`
+XXX: Do we use --save-dev
+XXX: What about creating an initial `package.json`
+XXX: How do other developers get the packages?
```

Comment:
I think it should be up to users to run `npm install` vs. Meteor somehow doing it for you, but we should try to figure out a way to warn users when their dependencies aren't installed? This could be as easy as something in `.meteor/local` which is a straight copy of `package.json`, and complains if it's different.